src/main/scala/com/fortysevendeg/translatebubble/ui/wizard/Layout.scala

```diff
@@ -61,6 +67,42 @@ trait Layout
     w[ImageView] <~ paginationItemStyle <~ vTag("position_%d".format(position))
```

Comment:
The parameter of `setTag` in Android SDK is a `Object` and we have `String` in `macroid-extra`. Maybe we should review that again